openapi_godoc_test.go

```diff
@@ -126,7 +126,7 @@ func TestGenerate(t *testing.T) {
 		ExternalDocs: ExternalDocs{Description: "Find out more", URL: "http://example.com"},
 	}
 	expectedResult := []byte(`{"components":{"responses":{"NotFound":{"content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}},"description":"The specified resource was not found"},"Unauthorized":{"content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}},"description":"Unauthorized"}},"schemas":{"Error":{"properties":{"error":{"type":"string"},"status":{"type":"string"}},"type":"object"},"Message":{"properties":{"content":{"description":"The contents of a message","example":"Hello world!","type":"string"}},"type":"object"}}},"externalDocs":{"description":"Find out more","url":"http://example.com"},"info":{"description":"A hello world API","title":"Hello API","version":"1.0.0"},"openapi":"3.0.0","paths":{"/":{"get":{"description":"Returns a hello message","operationId":"sayHello","responses":{"200":{"content":{"application/json":{"schema":{"$ref":"#/components/schemas/Message"}}},"description":"successful operation"}},"summary":"Say Hello","tags":["hello"]}}},"servers":[{"url":"http://localhost:8080"}],"tags":[{"description":"hello related apis","name":"hello"}]}`)
```

Comment:
Does it have to be here or Go supports some kind of assets/resources folder?

Reply:
Good question, i'm not sure... i'll take a look and if i find a better way submit in another PR.